packages/KStepper/KStepper.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div
+    :style="stepperStyle"
+    class="k-stepper d-flex"
+  >
+    <KStep
+      v-for="(step, idx) in steps"
+      :key="`k-step-${step.label.replace(' ', '-')}`"
+      :label="step.label"
+      :state="step.state"
+      :is-first="idx === 0"
+      :is-last="(steps.length - 1) === idx"
+    />
+  </div>
+</template>
+
+<script>
+import KStep from './KStep.vue'
+
+export default {
+  name: 'KStepper',
+  components: { KStep },
+  props: {
+    /**
+     * Array of steps to display
+     */
+    steps: {
+      type: Array,
+      required: true,
+      validator: (items) => !items.length || items.some(i => i.hasOwnProperty('label') && i.hasOwnProperty('state'))
+    },
+    maxWidth: {
+      type: String,
+      default: '900'
+    }
+  },
+  computed: {
+    stepperStyle: function () {
```

Comment:
We typically define computed variables like this, so unless a specific reason, can we change the format here as well?
```suggestion
    stepperStyle () {
```

Reply:
Fixed